js/Main.js

```diff
@@ -3,11 +3,4 @@ jQuery.get({url: baseUrl + "/config.yml", dataType: "text"})
             configData = jsyaml.load(data);
             initializeButtons();
         }
-);
-
-    
-(function(){
-    const params = getQueryParameter();
-    const selectedSim = params.get('sims');
-    document.title = `${Sims[selectedSim]} | ${document.title}`;
-}());
```

Comment:
we should leave this in, this fixed the tab title in the browser

Reply:
Ahh neet will add it again :)